src/main.cc

```diff
@@ -51,7 +51,8 @@ void Serialization(std::ostream *stream,
     *stream << std::flush;
 }
 
-int ProcessAST(const Config& config, std::unique_ptr<std::istream>& in, std::unique_ptr<std::ostream>& out) {
```

Comment:
👍 Yes, let's settle on format and I'll push the PR through at the end of a sprint when we've got no open work.